test/unit/basic.dev.test.js

```diff
@@ -26,11 +27,16 @@ describe('basic dev', () => {
       }
     })
     nuxt = new Nuxt(config)
-    await new Builder(nuxt).build()
+    builder = new Builder(nuxt)
```

Comment:
It's fine here, but acutally `build()` also returns `builder` object.

Reply:
@clarkdo gotcha.